Resources/views/Default/admin.html.twig

```diff
@@ -10,13 +10,6 @@
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', {'callback': 'fos.Router.setData'}) }}"></script>
     <script type="text/javascript">
-      //var CSClientId = '{{ clientId }}';
```

Comment:
I also changed it on stash in a same way its here after talk with Mark. Next time this file wont be updated, the one on stash will be only.